schemas/network.yml

```diff
@@ -48,3 +64,13 @@
       description: >
         Network outbound packets.
       example: 12
+    - name: total.bytes
+      type: long
+      description: >
+        Network Total bytes: Usually sum (inbound.bytes, outbound.bytes)
+      example: 368
+    - name: total.packets
+      type: long
+      description: >
+        Network Total packets: Usually sum (inbound.packets, outbound.packets)
+      example: 24
```

Comment:
I would phrase both of these more directly, e.g. "The sum of inbound.packets + outbound.packets", same for bytes.